esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2717,7 +2717,7 @@ bool CFileSprayEx::onFileList(IEspContext &context, IEspFileListRequest &req, IE
         if (!isEmptyString(fileNameMask))
         {
             const char* ext = pathExtension(sPath.str());
-            if (!strieq(ext, "cfg") && !strieq(ext, "log"))
+            if (ext && !strieq(ext, "cfg") && !strieq(ext, "log"))
```

Comment:
Either the condition is wrong or the message is wrong, since I think as coded this means the extension must be cfg, log, or none...